pom.xml

```diff
@@ -140,6 +140,12 @@
       <artifactId>grpc-all</artifactId>
       <version>0.9.0</version>
     </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-interop-testing</artifactId>
```

Comment:
You really shouldn't depend on grpc-interop-testing; it's only there for people to run the interop clients if they want. It shouldn't be hard to remove the dependency of `AbstractTransportTest`.